### ISSUE TITLE
Question/Suggestion  Waypoint system for docking smaller craft (or even larger craft) 

### ISSUE THREAD

**Issue by ExodistSKY1** [tsunamayo/Starship-EVO#2444]
QUESTION:  Are you planning on implementing a waypoint system for docking/ landing?

   An invisible guide block (a landing and take off block) that can be seen with the link tool selected. This invisible block would use the link tool to create a waypoint system for more refined and efficient Player/ AI landings and takeoffs.  

    For this to work more efficiently I think having a radial menu with the middle mouse button click that displays all available docks for your ship on the specific ship/station you can select one and your ship will automatically head towards the last linked waypoint and guide itself into a nice landing. 

taking off would be very similar where taking off via waypoint would have your ship follow the customized path

**Comment by tsunamayo** [comment]
yes maybe that is something that will be needed, I need to make more progress on ship in ship, this will come in time!
